addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <odoo><data>
 
-<template id='course_sidebar' inherit_id="website_slides.course_sidebar">
+<template name="Product" id='course_sidebar' inherit_id="website_slides.course_sidebar">
```

Comment:
Good idea to add a name! I would suggest to improve it a bit because Product is maybe a bit generic: maybe something like "Join Channel: Product Button" or something like that.